src/screens/RecoverySend/RecoverySendScreen.tsx

```diff
@@ -193,16 +196,16 @@ export class RecoverySendScreen extends Component<Props, State> {
       for (let tries = 0; tries < MAX_TRIES; tries++) {
         amountWithoutFee = amount - fee;
 
-        tx = await wallet.createTx({
+        const { tx: txHex, fee: feeSatoshi } = await wallet.createTx({
```

Comment:
I think we can have simply 
```
const { tx, fee: feeSatoshi } = (...)
```

instead of 
```
const { tx: txHex, fee: feeSatoshi } = (...)
```

in this case line 187 `let tx = '';` and line 207 is not necessary

Reply:
![Zrzut ekranu 2020-10-15 o 12 53 29](https://user-images.githubusercontent.com/67273035/96114108-75b49280-0ee5-11eb-83a7-4804aae34945.png)